src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/noc/documents/generators/NocLetterDetailsGenerator.java

```diff
@@ -2,23 +2,27 @@
 
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.bsp.common.model.document.Addressee;
```

Comment:
This class needs to be deleted as its superseded by AbstractLetterDetailsGenerator